### ISSUE TITLE
Building Projects: [Windows VM] Steam IDE cannot run the game if there's a pre_project_step.bat script in the project

### ISSUE THREAD

**Issue by KeeVeeGames** [YoYoGames/GameMaker-Bugs#5930]
### Description
No matter what's the content of the bat file, the very existence of it is making unable to Run (F5) the project:

```
C:\Users\...\GameMakerProjects\SteamBatBug\pre_project_step.bat 
The given key (platformConfig) was not present in the dictionary.
Igor complete.
elapsed time 00:00:00.1886729s for command "C:\ProgramData\GameMakerStudio2\Cache\runtimes\runtime-2024.4.1.202/bin/igor/windows/x64/Igor.exe" -j=8  -options="C:\Users\...\AppData\Local\GameMakerStudio2\GMS2TEMP\build.bff" -v -- Windows Run started at 05/23/2024 15:05:47
FAILED: Run Program Complete
```

Building as executable is working.

I did not test the other batch scripts variants available to run.

### Steps To Reproduce
1. Open the project in any Steam IDE 2024.2.0 onwards
2. Run it with the matching runtime

### Which version of GameMaker are you reporting this issue for?
IDE v2024.4.1.152 Runtime v2024.4.1.202

### Which operating system(s) are you seeing the problem on?
Windows 10.0.19045.0

### Which platform(s) are you seeing the problem on?
Windows

[e2a1f71a-d7ab-4b26-ae1a-f60213e13411](https://api.gamemaker.io/api/github/downloads/e2a1f71a-d7ab-4b26-ae1a-f60213e13411)

**Comment by YYDan** [comment]
Yes, I see the same - non-Steam IDE works just fine with that project.

**Comment by YYDan** [comment]
IDE v2024.2.0.132 Steam  Runtime v2024.2.0.163 gives the same result.

Prior to that, this build step was not supported (and so this project will run fine in Nov23 or LTS, etc.)

**Comment by Emc1923** [comment]
Verified fixed as of  IDE v2024.6.0.154 Steam  Runtime v2024.6.0.204
